src/containers/DatasetsContainer.jsx

```diff
@@ -0,0 +1,167 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Route } from 'react-router';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+import { fetchDataset, emptyDataset, createDataset, removeDataset, updateDataset }
+  from '../actions/dataset';
+import { emptyResource } from '../actions/resource';
+import DatasetList from '../components/DatasetList';
+import DatasetDetail from '../components/DatasetDetail/DatasetDetail';
+import ResourceDetail from '../components/ResourceDetail/ResourceDetail';
+
+import './datasets-container.scss';
+
+const mapStateToProps = state => ({
+  dataset: state.dataset,
+  datasets: state.datasets,
+  resource: state.resource,
+  schema: state.schema,
+  uiDataset: state.uiDataset,
+  uiResource: state.uiResource
+});
+
+const mapDispatchToProps = dispatch => bindActionCreators({
+  onFetch: fetchDataset,
+  onEmpty: emptyDataset,
+  onCreate: createDataset,
+  onRemove: removeDataset,
+  onUpdate: updateDataset,
+  onEmptyResource: emptyResource
+}, dispatch);
+
+class DatasetsContainer extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      resourceToDataset: props.resourceToDataset
+    };
+
+    this.handleResourceToDataset = this.handleResourceToDataset.bind(this);
+    this.emptyResourceToDataset = this.emptyResourceToDataset.bind(this);
+  }
+
+  handleResourceToDataset(resource) {
+    this.setState({
```

Comment:
Not sure but would't be better add "resourceToDataset" in the redux store instead of a state of a component?

Reply:
yes good point I wil do that now

Reply:
gedaan